clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2939,6 +2939,65 @@ def ExtractMemberOp : CIR_Op<"extract_member", [Pure]> {
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// UpdateMemberOp
+//===----------------------------------------------------------------------===//
+
+def UpdateMemberOp : CIR_Op<"update_member",
```

Comment:
To match the Extract above it might be better be Insert instead of Update.

Reply:
Updated.